### ISSUE TITLE
Seems it does not work in the latest version of remote play.

### ISSUE THREAD

**Issue by czt616** [r57zone/DualShock4-emulator#68]
There is a new version today. And I cannot use my keyboard anymore.

**Comment by r57zone** [comment]
Вы установили Vigem? 
У вас эмулируется устройство в Win + R - Joy.cpl (WirelessController)? 
У вас не работает джойстик в PSPlus или PS4RemotePlay?

**Comment by The-Sapphire-Starship** [comment]
It is not working for me either after the newest update
After doing Win + R - Joy.cpl it shows the following for me:
Bluetooth LE XINPUT compatible input device     OK
Wireless Controller     OK

**Comment by r57zone** [comment]
I have updated the latest version, try it 
https://github.com/r57zone/DualShock4-emulator/releases/

**Comment by The-Sapphire-Starship** [comment]
No change, my ps4 did update right before trying this new version (mentioning that just in case that would affect anything)
I did make sure ViGEmBus and Visual CPP Redistributable was up to date and as far as I can tell they are

**Comment by r57zone** [comment]
does the old version 1.8.1 still work?
https://github.com/r57zone/DualShock4-emulator/releases/tag/1.8.1

**Comment by The-Sapphire-Starship** [comment]
Nope, the only thing that happens is when I press the xbox button it noves from "Whats New" to the first item in the list in the ps4 home screen

**Comment by r57zone** [comment]
Can you run the emulator then this utility https://github.com/Nielk1/VSCView/releases/tag/v0.5.0.0  ? Add a DualShock 4 controller and theme, will the buttons move?

**Comment by The-Sapphire-Starship** [comment]
Yea, it all works fine with that
Something seems to be goings on
Could Sony be going something to detect that we are doing this and keep it from working?

**Comment by r57zone** [comment]
Maybe

**Comment by ChimiWei** [comment]
not working here as well, waiting & hoping for a fix  😢😢😢

**Comment by The-Sapphire-Starship** [comment]
same, after using the xbox controller with my ps4 for a while it is kinda difficult to use the ps4 controller, it is doable and I will if its the only option, but i do prefer using the xbox controller

**Comment by MerThe** [comment]
I have the same issue as well. Remote Play detects the controller and Xbox Button works as PS Button but ALL other buttons does not work at all. I tested it on Steam's controller settings and it works fine. It sees it as a DS4. Remote Play causes the problem. It is as if it knows the app is emulation and refuses the controller inputs.

**Comment by r57zone** [comment]
Sony probably changed something and or just out of harm closed this opportunity.

**Comment by redbulled** [comment]
Same here. Only "PS" button works fine

**Comment by r57zone** [comment]
Other Vigem projects seem to have a similar problem, similar to the blocking of Sony.

It seems that version PS Remote Play 5.50 does not have this problem and works, if anyone has an installer left, could you share, I could post this version here.

Also can support this tweet to Sony as well - https://twitter.com/r57zone/status/1634121668711833601

**Comment by r57zone** [comment]
Found an old version - [PS_Remote_Play_5.5.0.08250.zip](https://github.com/r57zone/DualShock4-emulator/files/10940634/PS_Remote_Play_5.5.0.08250.zip)

**Comment by The-Sapphire-Starship** [comment]
That is working on my end, thank you. It is slightly annoying having to put in a code everytime but I am very glad that it is at least working. Hopefully it was just a weird bug on their end and will be fixed

**Comment by MerThe** [comment]
Thanks for the link. I tried looking for an older version but could not find it.

**Comment by ChimiWei** [comment]
unfortunaly i only use it away from home :/

**Comment by e-t-l** [comment]
Also having this issue when PS Remote Play updated to Version 6.0.0.2240. 

Does the old version you linked ([5.5.0.08250](https://github.com/r57zone/DualShock4-emulator/files/10940634/PS_Remote_Play_5.5.0.08250.zip)) have some change so that it won't prompt for an update? Because that was my issue; Remote Play v5.5.something was working fine, until today it required an update before it would connect.

> Can you run the emulator then this utility https://github.com/Nielk1/VSCView/releases/tag/v0.5.0.0 ? Add a DualShock 4 controller and theme, will the buttons works?

^Is that a temporary fix to make DS4emu work with Remote Play v6, or was it just something to check that DS4emu is still working?

**Comment by r57zone** [comment]
Can't update be cancelled? If not, then in theory blocking the url in the Windows hosts file can help, but you need to find out this host using the program
@e-t-l nothing has changed with DS4Emu, any DualShock 4 emulation based on the Vigem driver is broken
https://learn.microsoft.com/en-us/sysinternals/downloads/tcpview

**Comment by e-t-l** [comment]
I rolled back to v5.5 and declined the update prompts, and it looks like it'll work but only on LAN, which is fine for my purposes (for everyone else, maybe check out doing a VPN from your traveling device to your Playstation's home network with an app like Tailscale or ZeroTier).

> nothing has changed with DS4Emu, any DualShock 4 emulation based on the Vigem driver is broken 

Oh I know this is Sony's fault, not yours. Hopefully the VigEm maintainers can figure out some fix, since I doubt Sony will change anything to help people not use their proprietary controllers lol. 

I haven't tried sniffing the traffic for the update domain, but unless they use a unique server just to push updates and nothing else, then blocking the connection to RemotePlay's update would likely block RemotePlay itself.

Best bet right now is if some modded cracks the v5.5 executable and spoofs a higher version number so Sony thinks it's already up-to-date.

**Comment by pancakeslp** [comment]
People seem to have found a work around for having remote play v5.5 appear as remote play v6
https://www.reddit.com/r/remoteplay/comments/11m5fu6/i_need_help_ps4macro_doesnt_work_anymore/jbgad6t/?context=3
use at your own risk

**Comment by e-t-l** [comment]
Great find, thanks for sharing! Given that the patch itself is open source, I'm not too worried.

**Comment by r57zone** [comment]
If it works don't forget to share, I'll post it in the releases, thanks

**Comment by MerThe** [comment]
The nethod pancakeslp shared works! Thanks for the find, mate. You are a savior.

**Comment by mrdeadlocked** [comment]
Going back to 5.5 worked for me as well

**Comment by e-t-l** [comment]
I'm trying to set this up on a new computer and I must be doing something wrong, because while it was working 100% perfectly on my old PC, it doesn't now. I have Remote Play v5.0.0 installed, patched with XeroPresence patcher to appear as v6.5. I have latest ViGEmBus (v1.2.2.0) and DS4emulator v2.0.1. Yet I am getting the exact same issue as described by @MerThe and @redbulled, where the only input Remote Play recognizes is the PS Button press. I have confirmed via both Joy.CPL and VSCView that the emulated DS4 controller is being recognized by the Windows system; just not by Remote Play. 

Can anyone help me figure out what setup step I might have missed? Because I can't remember what I did differently on my last PC that made it work flawlessly. I also tried installing RemotePlay v5.5 from the link above, but as you can see in that Reddit thread, the MSI installer no longer works...

**Comment by r57zone** [comment]
Try an alternative client https://sr.ht/~thestr4ng3r/chiaki/  , in the official Sony somehow closed emulated gamepads.

**Comment by e-t-l** [comment]
I'm gaming with Chiaki right now, actually. It's fine but it doesn't have all the awesome work you put into mapping DS4 touchpad presses/swipes.

It's weird because the patched Remote Play client was working for me (on my old PC) just last week. I can't figure out how to get it set up in the same way. I tried copying over the whole `C:\Program Files (x86)\Sony\PS Remote Play` and all related reg keys I could find on my old hard drive, but it wouldn't run* so something is missing.

*specifically, it kept throwing error 0x80001fff when trying to register the console in the client
